src/webview/webview2/mod.rs

```diff
@@ -392,6 +390,8 @@ window.addEventListener('mousemove', (e) => window.chrome.webview.postMessage('_
             let mut js = PWSTR::null();
             args.TryGetWebMessageAsString(&mut js)?;
             let js = take_pwstr(js);
+            // Winit 0.28 doesn't have manual hittest yet.
+            #[cfg(not(feature = "winit"))]
```

Comment:
we can just port the code of hittest here, it shouldn't be a problem

Reply:
There's also `window.begin_resize_drag`. And the hit_test function will need to call several DPI[ foreign functions](https://github.com/tauri-apps/tao/blob/dev/src/platform_impl/windows/util.rs#L318) which aren't included in windows crate.
Can you open an issue instead for future improvement?